### DISCUSSION TITLE
How could I make my character stay while Dialogue activating?

### DISCUSSION THREAD

**Discussion by ZigLLAP** [dialogic-godot/dialogic#2403] [Q&A]
I found that my character can move freely during the conversation, and clicking on other dialogue boxes will overwrite the current dialogue. Another issue is that after I set up Area2D, the character can start a conversation when contacting an object, but if I press the confirm key again in the same spot, the dialogue keeps refreshing. Is there a way to let a dialogue finish completely before starting again?

**Comment by mak448a** [comment]
Freeze him in the script and don't allow him to move by checking for a variable "movable" that you set to false when dialogic is run.
